app/main/views.py

```diff
@@ -67,16 +67,10 @@ def view(service_id):
     except HTTPError:
         flash({'api_error': service_id}, 'error')
         return redirect(url_for('.index'))
-
-    presented_service_data = {}
-    for key, value in service_data.items():
-        presented_service_data[key] = presenters.present(
-            value, content.get_question(key)
-        )
-
+    content.filter(service_data)
```

Comment:
We just discussed this. It's not thread safe because it changes the state of the content loader, which is shared.